Dangerfile

```diff
@@ -0,0 +1,54 @@
+# Show our appreciation.
+unless github.api.organization_member?('simp', github.pr_author)
+  message(':tada: Thanks for your contribution!')
+end
+
+# Defines a few ways of declaring a change as trivial.
+def declared_trivial
+  github.pr_title.match(/^(?:WIP: ?)?\(SIMP-MAINT/) ||
+  github.pr_body.match(/^#trivial/)
+end
+
+# Don't allow Work in Progress PRs to merge
+fail('PR is classed as Work in Progress') if github.pr_title.start_with? 'WIP:'
+
+# Reviewers are most effective at evaluating small change-sets as measured
+# in lines of code (LOC).  Less than 50 LOC is ideal, but 200 is more realistic
+# and still within acceptable limits.  After about 400 LOC reviewers tend
+# to skim over most of the code, effectively nullifying their review.  To
+# avoid this, disallow trivial change-sets over 200 LOC, and otherwise warn.
```

Comment:
There's a sort of irony in the LOC this explanatory comment block takes. :sunglasses: